pkg/resourcemanager/rediscaches/redis/rediscache_reconcile.go

```diff
@@ -50,16 +50,14 @@ func (rc *AzureRedisCacheManager) Ensure(ctx context.Context, obj runtime.Object
 				instance.Status.Message = err.Error()
 				return false, err
 			}
-			instance.Status.Message = resourcemanager.SuccessMsg
 			instance.Status.State = string(newRc.ProvisioningState)
```

Comment:
minor: I would put the state setting next to the message/provisioned/provisioning setting